File: XClip.Api/Models/batchItem.cs

using System;
using System.Collections.Generic;

namespace XClip.Api.Models
{
    public class batchItem
    {
        public Guid Id { get; set; }
        public int index { get; set; }
        public string start { get; set; }
        public string stop { get; set; }
        public string duration { get; set; }
        public List<int> tags { get; set; } 
    }
}

File: XClip.Web/js/models/batchItem.js
var batchItem = function(data) {
    this.index = null;
    this.start = null;
    this.stop = null;
    this.duration = null;
    this.tags = new Array();

    if (data != null && data != undefined) {
        this.index = data.index;
        this.start = data.start;
        this.stop = data.stop;
        this.duration = data.duration;
        this.tags = data.tags;
    }
};
